views-fieldset-tck/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id("io.micronaut.build.internal.views-module")
+}
+dependencies {
+    api(mnTest.micronaut.test.junit5)
+    api(projects.micronautViewsFieldset)
+    api(projects.micronautViewsCore)
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+    api(mnValidation.micronaut.validation)
+}
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
+    }
+}
```

Comment:
What is the reason for publishing this module?

Reply:
yes, we will use it in starter.